pkgs/top-level/all-packages.nix

```diff
@@ -26597,6 +26606,28 @@ with pkgs;
 
   thttpd = callPackage ../servers/http/thttpd { };
 
+  stalwart-mail_0_6 = stalwart-mail.overrideAttrs (old: rec {
+    pname = "stalwart-mail_0_6";
+    version = "0.6.0";
+    src = fetchFromGitHub {
+      owner = "stalwartlabs";
+      repo = "mail-server";
+      rev = "v${version}";
+      hash = "sha256-OHwUWSUW6ovLQTxnuUrolQGhxbhp4YqKSH+ZTpe2WXc=";
+      fetchSubmodules = true;
+    };
+    cargoDeps = old.cargoDeps.overrideAttrs (_: {
+      inherit src;
+      outputHash = "sha256-M4mAUI4hBso9rlSrRxpjO/DoWqKwg38kMAhb0E33onE=";
+    });
+    env = {
+      OPENSSL_NO_VENDOR = true;
+      ZSTD_SYS_USE_PKG_CONFIG = true;
```

Comment:
I think you can inherit those two lines by just overriding the two rocksdb settings:

```nix
env = old.env // {
  ROCKSDB_INCLUDE_DIR = /*...*/;
  ROCKSDB_LIB_DIR = /*...*/;
};
```

Reply:
Thank you :+1: